### PR TITLE
Refactor GeolocatorService.Reactive

### PR DESCRIPTION
**Pull request by charlesericlanglois**
GitHub Issue: #

## What is the current behavior?

GeolocatorService.Reactive is divided between GeolocatorService.Reactive.Uno.UI and GeolocatorService.Reactive.Shared

## What is the new behavior?

There is only one GeolocatorService.Reactive package

## Checklist

Please check if your PR fulfills the following requirements:

- [ ] Documentation has been added/updated
- [ ] Automated Unit / Integration tests for the changes have been added/updated
- [ ] Contains **NO** breaking changes
- [ ] Updated the Changelog
- [x] Associated with an issue

Use the GeolocatorService.Reactive package instead of GeolocatorService.Reactive.Uno.UI or GeolocatorService.Reactive.Shared

### CODE REVIEW

File: GeolocatorService.sln [hunk 1]
```diff
@@ -13,14 +13,12 @@ Project("{2150E333-8FDC-42A3-9474-1A3956D46DE8}") = "Library", "Library", "{FFE6
 EndProject
```
Comment (jeanplevesque) [inline GeolocatorService.sln:13]: The newly added files are good, but you also need to remove the ones that shouldn't be there. They need to be removed from the repository.

Comment (jeanplevesque) [inline GeolocatorService.sln:13]: You checked the "No breaking changes" box, but changing the structure of the packages **is a breaking change** and it needs to be documented.

**Review by jeanplevesque — CHANGES_REQUESTED** [review]

File: GeolocatorService.sln (hunk 1 above)
Comment (charlesericlanglois) [reply GeolocatorService.sln:13]: Done

Comment (charlesericlanglois) [reply GeolocatorService.sln:13]: I misunderstood that checkbox, let me know if my description works

Comment (jeanplevesque) [reply GeolocatorService.sln:13]: You also need to document the breaking change in BREAKING_CHANGES.md, you can take a look at this as an example: https://github.com/nventive/Chinook.DynamicMvvm/blob/main/BREAKING_CHANGES.md

**Review by jeanplevesque — CHANGES_REQUESTED** [review]

File: GeolocatorService.sln (hunk 1 above)
Comment (charlesericlanglois) [reply GeolocatorService.sln:13]: Done

File: src/GeolocatorService.Reactive/GeolocatorServiceExtensions.cs [hunk 2]
```diff
@@ -39,7 +38,7 @@ public static IObservable<Geocoordinate> GetAndObserveLocationOrDefault(this IGe
 		private static async Task GetInitialLocationOrDefault(this IGeolocatorService service, IObserver<Geocoordinate> observer, CancellationToken ct)
 		{
 			var isPermissionGranted = await service.RequestPermission(ct);
-			var currentLocationOrNull = isPermissionGranted ? await service.GetLocationOrDefault(ct) : null;
+			var currentLocationOrNull = isPermissionGranted ? await service.GetLocation(ct) : null;
```
Comment (jeanplevesque) [inline src/GeolocatorService.Reactive/GeolocatorServiceExtensions.cs]: Should this be changed?

Comment (charlesericlanglois) [reply src/GeolocatorService.Reactive/GeolocatorServiceExtensions.cs]: No, I just reverted it

**Review by jeanplevesque — APPROVED** [review]

**Pull request merged**